app/src/main/java/de/xikolo/App.kt

```diff
@@ -69,6 +79,10 @@ class App : Application() {
         configureRealm()
         configureDefaultSettings()
         configureWebView()
+
+        if (Build.VERSION.SDK_INT >= Feature.SHORTCUTS) {
```

Comment:
Doing this creates a problem for Android Studio. 
"Call requires API level 25(current min is 21)" 
I don't understand this, because we set the check for API level 25 in Feature.SHORTCUTS. Any ideas?

Reply:
Ben is right, everything in `Feature` should provide flags, which indicate if features are available or not. If Android Studio / Lint warns you, put this into a separate method (what you should do anyway) and use the [`TargetApi` annotation](https://developer.android.com/reference/android/annotation/TargetApi).